crates/rpc/src/method/simulate_transactions.rs

```diff
@@ -1741,8 +1741,13 @@ pub(crate) mod tests {
         );
     }
 
+    #[rstest::rstest]
+    #[case::v07(RpcVersion::V06)]
```

Comment:
that should probably be `case::v_06`...

Reply:
Thanks, fixed.

While working on these tests yesterday night I couldn't figure out why the test output had `case_v07` for the v06 test ;) Apparently I should have stopped there...